### PR TITLE
added access to less internal options

### PR DESCRIPTION
**Pull request by ferozco**
Improvement to existing type definition.
http://lesscss.org/usage/#programmatic-usage

### CODE REVIEW

File: less/less.d.ts [hunk 1]
```diff
@@ -72,6 +77,8 @@ declare namespace Less {
 }
 
 interface LessStatic {
+    options: Less.StaticOptions;
```
Comment (thasner) [inline less/less.d.ts]: i haven't used this before, but the docs seems to indicate setting `less = <options>`, not `less.options = <options>` http://lesscss.org/usage/#using-less-in-the-browser-options

Comment (thasner) [inline less/less.d.ts]: errr, just saw the [update to V2](http://lesscss.org/usage/#v2-upgrade-guide-browser-usage). seems good

**Comment by thasner** [comment]
👍

File: less/less.d.ts (hunk 1 above)
Comment (Ferezco) [inline less/less.d.ts]: Theres some weirdness with the way you configure less. As you pointed out the docs say that you have to declare a global variable called less prior to less being loaded. Once less has been loaded that options object is then stored within lessStatic

**Pull request merged**